shopify/resources/discount_code.py

```diff
@@ -0,0 +1,8 @@
+from ..base import ShopifyResource
+
+
+'''
+Discount Code for Price Rules API
+'''
```

Comment:
i would move this as a docstring of the `DiscountCode` class and use `"` instead of `'` ( 🚲 🏠 )